actors/builtin/market/market_actor.go

```diff
@@ -216,58 +205,35 @@ func (a Actor) PublishStorageDeals(rt Runtime, params *PublishStorageDealsParams
 			deal.Proposal.Provider = provider
 			deal.Proposal.Client = client
 
-			st.lockBalanceOrAbort(rt, client, deal.Proposal.ClientCollateral, ClientCollateral)
-			st.lockBalanceOrAbort(rt, client, deal.Proposal.TotalStorageFee(), ClientStorageFee)
-			st.lockBalanceOrAbort(rt, provider, deal.Proposal.ProviderCollateral, ProviderCollateral)
+			err, code := st.lockBalanceOrAbort(msm.escrowTable, msm.lockedTable, &deal.Proposal)
```

Comment:
Another nit about this name, since the function no longer takes in fees but a deal proposal it should be something explicitly referencing the deal like "lockDealCollateral"

Reply:
Have changed this to `lockClientAndProviderBalances`.